webapp/packages/plugin-data-viewer/src/DataViewerSettingsService.ts

```diff
@@ -45,6 +46,10 @@ export class DataViewerSettingsService extends Dependency {
     return this.settings.getValue('plugin.data-viewer.disableCopyData');
   }
 
+  get disableExportData(): boolean {
+    return this.settings.getValue('plugin.data-export.disabled');
+  }
```

Comment:
please add old alias:
```
      /** @deprecated Use settings instead, will be removed in 23.0.0 */
      createSettingsAliasResolver(this.settingsResolverService, this.settings, { 'plugin.data-export.disabled': 'plugin_data_export.disabled' }),
```
also rename `plugin.data-export.disabled` to `plugin.data-viewer.export.disabled` and add alias to `plugin.data-export.disabled`